ibm/service/power/data_source_ibm_pi_volume_test.go

```diff
@@ -21,6 +21,7 @@ func TestAccIBMPIVolumeDataSource_basic(t *testing.T) {
 				Config: testAccCheckIBMPIVolumeDataSourceConfig(),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttrSet("data.ibm_pi_volume.testacc_ds_volume", "id"),
+					resource.TestCheckResourceAttr("data.ibm_pi_volume.testacc_ds_volume", "replication_enabled", "false"),
```

Comment:
This is simple volume DS test.. could be replication enabled or not. We can remove this line.